backend/server/server.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/99designs/gqlgen/handler"
+	"github.com/underthebus/lazy-invoice/backend/graphql"
+)
+
+const defaultPort = "8080"
```

Comment:
```suggestion
const defaultPort = "8081"
```
Webpack dev server is already on 8080